spec/factories/whitehall_export/edition_factory.rb

```diff
@@ -21,7 +21,7 @@
     contacts { [] }
     revision_history do
       [
-        { "event" => "create", "state" => state, "whodunnit" => 1 },
+        build(:revision_history_event),
```

Comment:
This looks good, be nice to pass the state into it.

Reply:
could also pass in the created_at value from the edition?

Reply:
Good suggestion with passing `created_at` in however I don't think passing state in is a good idea, solely because the default revision_history_event mimics the first event created when an edition is freshly drafted. E.g event has the value of 'create' and not 'update'. 

Passing the state in would mean that you could have an event in the revision history that couldn't occur in reality. I hit this problem and decided we probably shouldn't pass the state in now some code realises on the accuracy of the version history. What do you think?